### PR TITLE
Fix missing ntp

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by imobach** [comment]
LGTM

File: src/clients/ntp-client_auto.rb [hunk 1]
```diff
@@ -76,9 +76,15 @@ def main
         #               cloning the just installed system
         if Mode.config && Stage.initial
           ntp_conf = "/etc/ntp.conf"
+          installed_ntp_conf = File.join(Installation.destdir, ntp_conf)
+          # if ntp is not installed at all we cannot copy it, so return empty values
+          if !::FileUtils.exist?(installed_ntp_conf)
+            Builtins.y2milestone("Ntp is not installed, so return empty hash")
```
Comment (kobliha) [inline src/clients/ntp-client_auto.rb]: Builtins :)?

Comment (jreidinger) [inline src/clients/ntp-client_auto.rb]: well, my goal is to make small fix. If I plan to use logger, then I probably ends up using auto client, which will be more work and deserve scrum item :)...or do I overlook typo?

Comment (kobliha) [inline src/clients/ntp-client_auto.rb]: No typo I'd know of. But just that we should omit using B*s. It's not a hard requirement though.

**Pull request merged**